lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1002,17 +1002,18 @@ def restart_all_services(self):
                 "sudo systemctl stop 'magma@*' 'sctpd' ;"
                 "sudo systemctl start magma@magmad",
             )
+            self._wait_for_pipelined_to_initialize()
+
+            EXTRA_WAIT_TIME_FOR_OTHER_SERVICES_SECONDS = 10
```

Comment:
Maybe we can collect the constants defined in this class and make the class attributes similar to how this is handled in `S1ApUtil` or `SubscriberUtil`